rdr_service/model/bq_questionnaires.py

```diff
@@ -1299,6 +1299,30 @@ class BQPDRPPIModuleFeedbackView(BQModuleView):
     __pk_id__ = ['participant_id', 'questionnaire_response_id']
     _show_created = True
 
+# PDR-861:  Add WEAR Consent module:
+class BQPDRWearConsentSchema(_BQModuleSchema):
+    """ WEAR Consent Module """
+    _module = 'wear_consent'
+    _force_boolean_fields = (
+        'timeofday',
+        'HelpMeWithConsent_Name'
```

Comment:
These are the only two `survey_question_option` codes I found after the codebook import that were `SurveyQuestionCodeType.TEXT`.   The others are all `SurveyQuestionCodeType.RADIO`